aws-lambda-java-log4j2/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.amazonaws</groupId>
     <artifactId>aws-lambda-java-log4j2</artifactId>
-    <version>1.6.0</version>
+    <version>1.6.1-SNAPSHOT</version>
```

Comment:
we don't release "SNAPSHOT" artifacts to maven, could you change it to `1.6.1`?

Reply:
Sure, wasn't aware it needs to be manually done. I usually use maven-release-plugin.